athena-federation-sdk-tools/src/main/java/com/amazonaws/athena/connector/validation/LambdaMetadataProvider.java

```diff
@@ -100,7 +101,7 @@ public static ListTablesResponse listTables(String catalog,
     log.info("Submitting ListTablesRequest with ID " + queryId);
 
     try (ListTablesRequest request =
-                 new ListTablesRequest(identity, queryId, catalog, schema)) {
+                 new ListTablesRequest(identity, queryId, catalog, schema, null, UNLIMITED_PAGE_SIZE_VALUE)) {
```

Comment:
we should add a todo here that the validator should ensure the connector supports pagination. We may need to wait on this until you update the connectors.

Reply:
added TODO